sdk/servicebus/service-bus/CHANGELOG.md

```diff
@@ -1,12 +1,14 @@
 # Release History
 
-## 1.1.7 (2020-05-06)
+## 1.1.7 (2020-05-13)
 
 - Relaxes the scheme check for the endpoint while parsing the connection string.
   This allows "\<anything\>://" as the scheme as opposed to the "sb://" scheme suggested by the connection string in the portal.
   Fixes [bug 7907](https://github.com/Azure/azure-sdk-for-js/issues/7907).
 - Provided down-leveled type declaration files to support older TypeScript versions 3.1 to 3.6.
   [PR 8515](https://github.com/Azure/azure-sdk-for-js/pull/8515)
+- Updates `@azure/amqp-common` to version 1.0.0-preview.15, fixing an issue with OperationTimeoutError's not being 
+  considered retryable.
```

Comment:
```suggestion
- Updates `@azure/amqp-common` to version 1.0.0-preview.15, fixing an issue with OperationTimeoutError's not being considered retryable.
```

Reply:
(Just stopping this newline to show up at the github release page)

Reply:
I'll merge this after the test run completes :)